docs/examples/affinity/cookie/README.md

```diff
@@ -1,28 +1,29 @@
-# Sticky Session
+# Sticky sessions
 
-This example demonstrates how to achieve session affinity using cookies
+This example demonstrates how to achieve session affinity using cookies.
 
 ## Deployment
 
-Session stickiness is achieved through 3 annotations on the Ingress, as shown in the [example](ingress.yaml).
+Session stickyness can be configured using the following annotations. Make sure to prefix their names with `nginx.ingress.kubernetes.io/` when setting them.
```

Comment:
Why are you changing `stickiness` to `stickyness`?

Reply:
Typo, will fix. :smile: